public/app/features/admin/UserProfile.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { PureComponent, FC } from 'react';
+import appEvents from 'app/core/app_events';
+// import { dateTime } from '@grafana/data';
+import { UserDTO, CoreEvents } from 'app/types';
+import { cx, css } from 'emotion';
+import { ConfirmButton, Input, InputStatus } from '@grafana/ui';
+
+// const defaultTimeFormat = 'dddd YYYY-MM-DD HH:mm:ss';
+
+interface Props {
+  user: UserDTO;
+
+  onUserUpdate: (user: UserDTO) => void;
+  onUserDelete: (userId: number) => void;
+  onUserDisable: (userId: number) => void;
+}
+
+interface State {
+  isLoading: boolean;
+}
+
+export class UserProfile extends PureComponent<Props, State> {
+  handleUserDelete = () => {
+    const { user, onUserDelete } = this.props;
+    appEvents.emit(CoreEvents.showConfirmModal, {
+      title: 'Delete user',
+      text: 'Are you sure you want to delete this user?',
+      yesText: 'Delete user',
+      icon: 'fa-warning',
+      onConfirm: () => {
+        onUserDelete(user.id);
+      },
+    });
```

Comment:
Not sure about this one, couldn't you use the React Modal instead?

Reply:
We do not have a confirm modal as reusable pattern in react yet. So think this is fine for now as it would be a lot of code & state to use the React modal directly here.